src/Sylius/Bundle/ProductBundle/Behat/ProductContext.php

```diff
@@ -94,6 +94,10 @@ public function thereAreProducts(TableNode $table)
                 $this->configureProductPricingCalculator($product, $data);
             }
 
+            if (isset($data['average rating'])) {
```

Comment:
Average rating should be calculated in Behat scenarios  too. So we should not have anything like that here.